cluster/gce/gci/configure.sh

```diff
@@ -436,6 +443,56 @@ function install-docker {
   rm -rf /var/lib/apt/lists/*
 }
 
+# If we are on ubuntu we can try to install containerd
+function install-containerd {
```

Comment:
return early if UBUNTU_INSTALL_CONTAINERD_VERSION and UBUNTU_INSTALL_RUNC_VERSION are unset so we don't do package install work unnecessarily?

Reply:
yep, can do that when i get this version working

Reply:
Done